src/stirling/demo_apps/go_http/server/simple_http_server.go

```diff
@@ -68,13 +68,6 @@ func randStringRunes(n int) string {
 	return string(b)
 }
 
-func min(x, y int64) int64 {
-	if x < y {
-		return x
-	}
-	return y
-}
-
```

Comment:
A generic form of the [min function](https://pkg.go.dev/builtin#min) is built-in now. Since this file isn't compiled with `go_cross_binary`, we can depend on this stdlib support.